### PR TITLE
simplify new chapter

### PR DESCRIPTION
**Pull request by mcalmer**
Reduce new chapter to a description about how to import thrid party certificates and point
to another chapter for the rest of the process.

@Loquacity This is a draft. I think we need to add links to the other chapters and sections and maybe we want to rename and/or move the chapter .

Feel free to merge into your branch if you like the idea and continue with modifications.

### CODE REVIEW

**Comment by mcalmer** [comment]
Now it is ready.

File: modules/administration/pages/ssl-certs.adoc [hunk 1]
```diff
@@ -312,60 +312,44 @@ If you use bootstrap, remember to also update your bootstrap scripts to reflect
 
 
 
-== Replace Existing CA and Server Certificates
+== Replace Existing CA and Server Certificates with third party certificiate
 
 You can replace working certificates on an existing {productname} installation with a new third party certificate.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: Drive-by edit:
```suggestion
You can replace active certificates on your {productname} installation with a new third party certificate.
```

File: modules/administration/pages/ssl-certs.adoc [hunk 2]
```diff
@@ -312,60 +312,44 @@ If you use bootstrap, remember to also update your bootstrap scripts to reflect
...
 In this case, you can replace the old CA certificate RPM with the new one, and update the database.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: Drive-by edit:
```suggestion
To replace the certificates, you can replace the installed CA certificate RPM with a new RPM containing the third party certificate, and then update the database.
```

File: modules/administration/pages/ssl-certs.adoc [hunk 3]
```diff
@@ -312,60 +312,44 @@ If you use bootstrap, remember to also update your bootstrap scripts to reflect
...
 
+This procedure is similar to the one described in "Create and Replace CA and Server Certificates" with the difference, that we import the certificates generated by an external PKI.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: proper link wanted. we must create a section label first.

**Comment by Loquacity** [comment]
i'm going to merge this into my branch so we can keep working on it.
